proto/src/determined/user/v1/user.proto

```diff
@@ -28,6 +29,8 @@ message User {
 message PatchUser {
   // Name to display in the web UI.
   string display_name = 2;
+  // Updated hashed and salted password.
+  google.protobuf.StringValue password = 3;
```

Comment:
i see this is indeed a password hash, so can we call the variable `password_hash`